shared/constants/platform-specific/index.native.tsx

```diff
@@ -428,20 +423,6 @@ export const initPlatformListener = () => {
   C.useDaemonState.subscribe((s, old) => {
     if (s.handshakeVersion === old.handshakeVersion) return
 
-    // loadStartupDetails finished already
```

Comment:
i believe we don't need this part at all, this can happen entirely async and it shouldn't change anything